### PR TITLE
refactor: navbar component

### LINKED ISSUE #10: Vue Component: Navbar - GSoC'22
Navbar was implemented in project in [PR#4](https://github.com/CircuitVerse/cv-frontend-vue/pull/4)

Improvements required are -
- Convert Options API to Composition API
- Internationalization
- Fix CSS refactoring

### PR DESCRIPTION
**Pull request by devartstar**
Fixes #10 

Improvements required are -
- [x] Convert Options API to Composition API
- [x] Internationalization
- [x] Fix CSS refactoring

### CODE REVIEW

**Comment by devartstar** [comment]
Completed Localization of Navbar -
Data in the Navbar component are displayed from `NavbarData.json` and `UserData.json` files.
The value of this data matches the key of the locale(en/hi).json file and its values are displayed.

> I have kept the initial locale as Hindi to check in preview.

**Comment by devartstar** [comment]
@abstrekt @ayan-biswas0412 can you please review this PR.

File: src/components/Navbar/Navbar.vue [hunk 1]
```diff
@@ -33,30 +31,21 @@ import userDropdownItems from './UserData.json'
...
-    },
+import { ref } from '@vue/reactivity'
+import { onMounted } from '@vue/runtime-core'
+
+const navbarLogo = ref('logo')
+const minWidthToShowSidebar = ref(992)
+const isUserSignedIn = ref(false)
+const showSidebar = ref(false)
+showSidebar.value =
+    window.innerWidth < minWidthToShowSidebar.value ? true : false
```
Comment (abstrekt) [inline src/components/Navbar/Navbar.vue:42]: can we just use CSS media queries for what we are trying to achieve here?

File: src/components/Navbar/UserData.json [hunk 2]
```diff
@@ -1,19 +1,19 @@
 [
```
Comment (abstrekt) [inline src/components/Navbar/UserData.json:1]: Let's move all static, constants to `/constants`

File: src/components/Navbar/NavbarData.json [hunk 3]
```diff
@@ -1,54 +1,54 @@
 [
```
Comment (abstrekt) [inline src/components/Navbar/NavbarData.json:1]: Let's move all static, constants to `/constants`

**Review by abstrekt — APPROVED** [review]

File: src/components/Navbar/UserData.json (hunk 2 above)
Comment (devartstar) [reply src/components/Navbar/UserData.json:1]: Done

**Pull request merged**